src/controllers/features.controller.ts

```diff
@@ -51,17 +56,21 @@ export class FeaturesController {
   })
   public async get(): Promise<Response> {
     this.logger.debug('[get] started');
-    let retorno = [new FeaturesDbDataModel()];
+    let features = [new FeaturesDbDataModel()];
     let responseCode = this.HTTP_ERROR;
     try {
-        retorno = await this.featuresDatService.getAll();
+        features = await this.featuresDatService.getAll();
+        const termsIdx = features.findIndex((feature) => feature.name === 'terms_and_conditions');
```

Comment:
What if there is no `terms_and_conditions` flag? Or more than one but the first "found" disabled?